app/policies/profile_policy.rb

```diff
@@ -17,7 +17,9 @@ def update?
   # Only show hosts in our user account
   class Scope < ::ApplicationPolicy::Scope
     def resolve
-      only_matching_account
+      return scope.where('1=0') if user.account_id.blank?
+
+      scope.where(account_id: [nil, user.account_id])
```

Comment:
I'm not overly convinced about this, but we need to be able to fetch the canonical profiles somehow

Reply:
Agreed - it's not very intuitive; if we added a `canonical_profile_id` attribute it would make a lot more sense IMO.